maskinporten_api/maskinporten_client.py

```diff
@@ -12,8 +12,9 @@
 class MaskinportenClient:
     def __init__(self, env):
         p12_encoded = get_secret(f"/dataplatform/maskinporten/origo-certificate-{env}")
+        password = get_secret("/dataplatform/maskinporten/origo-certificate-password")
```

Comment:
Forskjellig passord for de to sertifikatene, så trenger `-{env}` her og?

Reply:
Husk at vi skal kunne utstede klienter til både `test` og `prod` i Maskinporten fra vårt `prod`-miljø :)

Reply:
Ah! Trodde det var samme passord jeg. I fix!

Reply:
> Husk at vi skal kunne utstede klienter til både `test` og `prod` i Maskinporten fra vårt `prod`-miljø :)

Jepp, `env` her er Maskinporten sitt miljø.